10-ambarom-vignette.Rmd

```diff
@@ -305,14 +307,14 @@ covid_educ_ests %>%
               columns = c("p_mediumchange", "p_mediumchange_se")) %>%
   tab_spanner(label = "Cut ties with school",
               columns = c("p_noschool", "p_noschool_se")) %>%
-  fmt_number(decimals = 1)
+  fmt_number(decimals = 1) 
```

Comment:
FYI the table is off the page in PDF.

Reply:
I don't know how to fix this. It looks like `tab_style()` edits to gt tables don't show up in PDF?

* https://github.com/rstudio/gt/issues/119
* https://github.com/rstudio/gt/issues/1363